tests/rest_api/setup

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+# export environment variables
+export MINIO_HOST="http://127.0.0.1:9000"
+export MINIO_ACCESS_KEY="minio_access_key"
+export MINIO_SECRET_KEY="minio_secret_key"
+export DATA_PATH="tests/cypress/integration/actions_tasks/assets/case_65_manifest/"
+export MINIO_ALIAS="local_minio"
+export PRIVATE_BUCKET="${MINIO_ALIAS}/private"
+export PUBLIC_BUCKET="${MINIO_ALIAS}/public"
+export TEST_BUCKET="${MINIO_ALIAS}/test"
+echo "Environment variables have been exported"
+
+# run docker containers
+docker-compose -f docker-compose.yml -f docker-compose.dev.yml -f components/analytics/docker-compose.analytics.yml -f tests/rest_api/docker-compose.minio.yml up -d --build
+echo "All containers have been runned"
+
+# create MINIO buckets with data
+MC_DIR="${HOME}/Downloads"
+MC_PATH="${MC_DIR}/mc"
+if [ ! -e ${MC_PATH} ]
+then
+    echo "Downloading mc tool"
+    wget -P ${MC_DIR} https://dl.min.io/client/mc/release/linux-amd64/mc
+    chmod +x ${MC_PATH}
+    alias mc=${MC_PATH}
```

Comment:
Such an alias could potentially conflict with popular tool Midnight Commander. I would suggest just use other way for using Minio Client, example:
```
${MC_PATH} alias set ${MINIO_ALIAS} ${MINIO_HOST} ${MINIO_ACCESS_KEY} ${MINIO_SECRET_KEY}
```